src/RepoDriver.sol

```diff
@@ -0,0 +1,508 @@
+        emit RepoOwnerUpdateRequested(repoId, forge, name);
+    }
+
+    /// @notice Builds the AnyApi generic `bytes` fetching request payload.
+    /// It instructs the operator to fetch the current owner of the repository.
+    /// @param forge The forge where the repository is stored.
+    /// @param name The name of the repository.
+    /// @return payload The AnyApi request payload.
+    function _requestPayload(Forge forge, bytes memory name)
+        internal
+        view
+        returns (bytes memory payload)
+    {
+        // slither-disable-next-line uninitialized-local
+        BufferChainlink.buffer memory buffer;
+        buffer = BufferChainlink.init(buffer, 256);
+        buffer.encodeString("get");
+        buffer.encodeString(_requestUrl(forge, name));
+        buffer.encodeString("path");
+        buffer.encodeString(_requestPath());
+        return buffer.buf;
+    }
+
+    /// @notice Builds the URL for fetch the `FUNDING.json` file for the given repository.
+    /// @param forge The forge where the repository is stored.
+    /// @param name The name of the repository.
+    /// @return url The built URL.
+    function _requestUrl(Forge forge, bytes memory name)
+        internal
+        pure
+        returns (string memory url)
+    {
+        if (forge == Forge.GitHub) {
+            return string.concat(
+                "https://raw.githubusercontent.com/", string(name), "/HEAD/FUNDING.json"
+            );
+        } else if (forge == Forge.GitLab) {
+            return string.concat("https://gitlab.com/", string(name), "/-/raw/HEAD/FUNDING.json");
+        } else {
+            revert("Unsupported forge");
+        }
+    }
+
+    /// @notice Builds the JSON path inside `FUNDING.json` where the repository owner is stored.
+    /// @return path The comma-separated JSON path.
+    function _requestPath() internal view returns (string memory path) {
+        // slither-disable-next-line uninitialized-local
+        string memory chainName;
+        if (block.chainid == CHAIN_ID_ETHEREUM) chainName = CHAIN_NAME_ETHEREUM;
+        else if (block.chainid == CHAIN_ID_GOERLI) chainName = CHAIN_NAME_GOERLI;
+        else revert("Unsupported chain");
+        return string.concat("drips,", chainName, ",ownedBy");
+    }
+
+    /// @notice Updates the repository owner. Callable only by the AnyApi operator.
+    /// @param requestId The ID of the AnyApi request.
+    /// Must be the same as the request ID generated when requesting an owner update,
+    /// this function will update the repository that was requested back then.
+    /// @param ownerRaw The new owner of the repository. Must be a 20 bytes long address.
+    function updateRepoOwnerByAnyApi(bytes32 requestId, bytes calldata ownerRaw) public {
+        RepoDriverAnyApiStorage storage storageRef = _repoDriverAnyApiStorage();
+        require(msg.sender == address(storageRef.operator), "Callable only by the operator");
```

Comment:
Here, we could also use the trusted forwarder but I guess chainlink operators won't need it.  :)

Reply:
No, we shouldn't do that. Especially since there may be a possibility to trick the Chainlink operator to call Caller as a part of the AnyApi response and grant the attacker full rights to act as the operator.